lib/sqlalchemy/sql/selectable.py

```diff
@@ -6742,7 +6783,7 @@ def where(
 
         """
         e = self._clone()
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325